static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.jsx

```diff
@@ -26,7 +26,7 @@ describe('TeamIssuesBreakdown', () => {
       expect(screen.getByText(status)).toBeInTheDocument();
     }
 
-    expect(screen.getByText('javascript')).toBeInTheDocument();
+    expect(await screen.findByText('javascript')).toBeInTheDocument();
```

Comment:
was this wrong before?

Reply:
for some reason switching things to useApiQuery requires findBy instead of getBy.